src/java/picard/util/IntervalListScatterer.java

```diff
@@ -107,7 +110,7 @@ public int compare(final Interval o1, final Interval o2) {
                         } else {
                             // Push this interval into the next scatter; re-inject it into the queue, then advance the scatter.
                             intervalQueue.addFirst(interval);
-                            accumulatedIntervalLists.add(runningIntervalList);
+                            accumulatedIntervalLists.add(runningIntervalList.uniqued());
```

Comment:
George and I read the code carefully and concluded that .uniqued() it is only called once per output IntervalList. So, no.